### PR TITLE
update hammerhead

### PR DESCRIPTION
**Pull request by LavrovArtem**

### CODE REVIEW

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
ping @helen-dikareva

File: src/client/automation/playback/type/index.js [hunk 1]
```diff
@@ -25,7 +25,9 @@ var SPECIAL_KEYS    = testCafeCore.KEY_MAPS.specialKeys;
 export default class TypeAutomation {
     constructor (element, text, typeOptions) {
         this.element = TypeAutomation.findTextEditableChild(element) || element;
-        this.text    = text.toString();
+
+        // eslint-disable-next-line no-restricted-properties
+        this.text = text.toString();
```
Comment (helen-dikareva) [inline src/client/automation/playback/type/index.js]: We can avoid so many comments for linter in this file. Please rename `this.text` -> `this.typingText`

File: src/client/ui/select-element.js [hunk 2]
```diff
@@ -94,7 +94,8 @@ function createOption (realOption, parent) {
     var isOptionDisabled = realOption.disabled || domUtils.getTagName(realOption.parentElement) === 'optgroup' &&
                                                   realOption.parentElement.disabled;
 
-    option.textContent = realOption.text;
+    // eslint-disable-next-line no-restricted-properties
+    nativeMethods.nodeTextContentSetter.call(option, realOption.text);
```
Comment (helen-dikareva) [inline src/client/ui/select-element.js]: Why we do not use native method to get `realOption's` text ?

Comment (LavrovArtem) [reply src/client/ui/select-element.js]: We override the 'text' property descriptors only for anchor and script element.

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Pull request merged**